ngraph/frontend/onnx_import/src/onnx_import/op/gru.cpp

```diff
@@ -20,7 +20,7 @@
 #include "default_opset.hpp"
 #include "gru.hpp"
 #include "ngraph/builder/split.hpp"
-#include "ngraph/frontend/onnx_import/core/null_node.hpp"
+#include "core/null_node.hpp"
```

Comment:
why `onnx_import` is removed here?

Reply:
This is internal source file which has access to `onnx_import` folder.
For example "default_opset.hpp" is also located in `onnx_import` folder.

From my point of view it is valid change, or we should change this for all headers (in any way I propose to make it in separate PR, I try to change as less as possible files).

Reply:
I think that for both include and source files we need the same approach because `core/null_node.hpp` is a public file and should be included as `onnx_import/core/null_node.hpp`. I am ok to do this in a separate PR.

Reply:
Agree, but when I tried to do it, I faced that we have this problem in several places.

Reply:
All parts were fixed.